settings.gradle

```diff
@@ -9,6 +9,20 @@ pluginManagement {
 }
 // END Added for Android
 
+plugins {
```

Comment:
What's going on here?

Reply:
Fixes the following deprecation warning during autogenerate task:

```
Java toolchain auto-provisioning enabled, but no java toolchain repositories declared by the build. Will rely on the built-in repository. This behaviour has been deprecated and is scheduled to be removed in Gradle 8.0. In order to declare a repository for java toolchains, you must edit your settings script and add one via the toolchainManagement block. See https://docs.gradle.org/7.6.4/userguide/toolchains.html#sec:provisioning for more details.
```